src/Client.php

```diff
@@ -439,6 +467,20 @@ public function send($method, $payload, $path, $params = [], array $headers = []
     }
 
     /**
+     * @param Request $request
+     * @param Response $response
+     * @param string $type
+     *
+     * @return File
+     */
+    protected function parsePdf(Request $request, Response $response, $type)
```

Comment:
why is this method named `parsePdf` ? nothing in it's code defines its working with PDF... so it can be any other type, not?